src/js/tooltip.js

```diff
@@ -55,9 +59,20 @@ class Tooltip {
 		// Do you render as soon as possible?
 		if (this.opts.showOnConstruction) {
 			this.show();
+
+            if(this.opts.closeAfter) {
+                this.closeAfter(this.opts.closeAfter);
+            }
 		}
+		else {
+            if(this.opts.showAfter) {
```

Comment:
The indentation is off in a few of the files. I believe this is due to using spaces instead of tabs. Could you update the indentation to be tabs please? Thanks.